### PR TITLE
Enable support for Linux Mint 18 with linux_install.sh

### PR DESCRIPTION
**Pull request by Chris-Johnston**
When running `qmk_install.sh` on my Linux Mint 18 machine, I encountered the following error:

```console
$ ./qmk_install.sh 
Sorry, we don't recognize your OS. Help us by contributing support!

https://docs.qmk.fm/contributing.html
```

`qmk_install.sh` was correctly running the `linux_install.sh` script, but that wasn't correctly identifying my distro as being compatible.

Unlike the results that I see on my Ubuntu VM, when I view the contents of `/etc/os-release`, I do not see `debian`, like the script expects. Here are the relevant contents of this file:

```console
$ grep ID /etc/os-release 
ID=linuxmint
ID_LIKE=ubuntu
VERSION_ID="18.3"
```

Surely this change will impact other ubuntu-derived linux distros also? I would appreciate any feedback if anyone is able to test on another distro.

I have verified that running this modified script will begin installing software packages. Before this change, I had previously installed QMK by copying/pasting these commands into my terminal.

```console
$ ./qmk_install.sh 
Hit:1 http://....
```

### CODE REVIEW

File: util/linux_install.sh [hunk 1]
```diff
@@ -38,5 +38,5 @@ fi
 else
 	echo "Sorry, we don't recognize your OS. Help us by contributing support!"
 	echo
-	echo "https://docs.qmk.fm/contributing.html"
+	echo "https://docs.qmk.fm"
```
Comment (drashna) [inline util/linux_install.sh]: Actually, could you use this link: https://docs.qmk.fm/#/contributing

**Comment by drashna** [comment]
@skullydazed This look good?

**Comment by skullydazed** [comment]
Thanks!

**Comment by Chris-Johnston** [comment]
Thanks for the PR review, I'm happy to contribute to the firmware behind 3 of my keyboards

**Pull request merged**